main.cpp

```diff
@@ -56,10 +56,23 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+    int velocity =98;
+    int velocityMax=127;
+    int velocityMin=64;
+
+    double height = 1.83;
+    double width =2.50;
+    double length= 4.583;
+
+    float pi =3.1415926f;
+    float gravity = 6.67430f;
+    float gain = 1.0f;
+
+    bool off =false;
+    bool startGame= true;
+    bool gameOver = false;
     
-    
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(number,velocity,velocityMax,velocityMin,height,width,length,pi,gravity,gain,off,startGame,gameOver); //passing each variable declared to the ignoreUnused() function
```

Comment:
It's important to try to make your code as readable as possible for other people looking at it.
fix the spacing around your `=` so it is consistent.  `double height = 1.83;` has the correct spacing around the `=`
add a space after the `,` so your variable names being passed to `ignoreUnused` aren't all clumped together.
- [ ] revise.